pypesto/optimize/optimizer.py

```diff
@@ -623,3 +629,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class ScipyDifferentialEvolutionOptimizer(Optimizer):
+    """
+    Global optimization using scipy's differential evolution optimizer.
+    Package homepage: https://scipy.org
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 1000}
+        self.options = options
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        bounds = []
+        for iNumBounds in range(0, len(lb)):
```

Comment:
please use snake case, i.e. only lower case and underscores for separation, for variable and function names. camel case only for classes and interfaces https://realpython.com/python-pep8/